src/librustc_metadata/schema.rs

```diff
@@ -531,3 +530,9 @@ impl_stable_hash_for!(struct GeneratorData<'tcx> { layout });
 // Tags used for encoding Spans:
 pub const TAG_VALID_SPAN: u8 = 0;
 pub const TAG_INVALID_SPAN: u8 = 1;
+
+#[derive(RustcEncodable, RustcDecodable)]
+pub struct EncodedExportedSymbols {
```

Comment:
This should have a comment on it, that it's used to avoid adding a `'tcx` parameter to `CrateRoot` (which I'm not even sure is a problem, if covariant, we'd just store it as `'static`).